.gitignore

```diff
@@ -1,4 +1,5 @@
 .*
 !.yamllint
 !.ansible-lint
+test/
```

Comment:
Did we mean to add this to the .gitignore?

Reply:
that was where I was dumping my local env file, ssh keys for a test gerrit  and passwords.ini file.